projects/exui-common-lib/src/lib/gov-ui/components/hmcts-pagination/hmcts-pagination.component.html

```diff
@@ -0,0 +1,44 @@
+<pagination-template #p="paginationApi" [id]="id" [maxSize]="maxSize" (pageChange)="pageChange.emit($event)"
+  (pageBoundsCorrection)="pageBoundsCorrection.emit($event)">
+
+  <nav class="hmcts-pagination" id="pagination-label">
+    <p class="govuk-visually-hidden" aria-labelledby="pagination-label">Pagination navigation</p>
+    <ul class="hmcts-pagination__list" *ngIf="!(p.pages.length <= 1)">
+
+      <li class="hmcts-pagination__item  hmcts-pagination__item--prev" [class.disabled]="p.isFirstPage()">
+        <a [routerLink]="" class="hmcts-pagination__link" *ngIf="1 < p.getCurrent()" (keyup.enter)="p.previous()"
+          (click)="p.previous()">Previous <span class="govuk-visually-hidden">set of pages</span></a>
+        <span *ngIf="p.isFirstPage()">
```

Comment:
@paulhowes-HMCTS - is there a way to avoid binding a function to the `*ngIf`? Maybe the function is trivial but it will be executed repeatedly.

Reply:
I did wonder that myself, however the component utilises ngx-pagination and the getCurrent() method is encapsulated in that component. If you have any suggestions on attempting a workaround I'm all ears!

Reply:
I was going off the following comment on the https://github.com/michaelbromley/ngx-pagination page: 
The key thing to note here is #p="paginationApi".

I would definitely use a local property for values that were readily accessible, however in this case I decided not to spend an excessive amount of time on it and would go with the instructions...

Reply:
Ah, that's fair enough then - I see that's how it's done in the example.